src/components/shared/VirtualList.js

```diff
@@ -152,48 +195,87 @@ VirtualListInner.propTypes = {
   columnIndex: PropTypes.number.isRequired,
 };
 
-class VirtualList extends PureComponent {
-  constructor(props) {
+type VirtualListProps = {|
+  +itemHeight: CssPixels,
+  +className: string,
+  +renderItem: RenderItem,
+  +items: *[],
+  +focusable: boolean,
+  +specialItems: *[],
+  +onKeyDown: KeyboardEvent => void,
+  +onCopy: Event => void,
+  +disableOverscan: boolean,
+  +columnCount: number,
+|};
+
+type Geometry = {
+  outerRect: DOMRect | ClientRect,
```

Comment:
why ClientRect ? Maybe I miss something but I think it comes solely from `getBoundingClientRect` so it's only a `DOMRect` ?

Reply:
If we don't need x or y, maybe we should just use a ClientRect.